src/web/handlers/LongPressGestureHandler.ts

```diff
@@ -104,7 +104,7 @@ export default class LongPressGestureHandler extends GestureHandler {
     if (this.minDurationMs > 0) {
       this.activationTimeout = setTimeout(() => {
         this.activate();
-      }, this.minDurationMs);
+      }, this.minDurationMs) as unknown as number;
```

Comment:
Same

Reply:
Done in [f6a46e5](https://github.com/software-mansion/react-native-gesture-handler/pull/2873/commits/f6a46e59bf7ebe489416d68fc2700e18d5bb3d05)